### PR TITLE
enhance: Improve GetVectorById of Sparse Float Vector

### PR DESCRIPTION
**Pull request by zhengbuqian**
issue: #29419

* sparse float vector to support raw data mmap

For get vector from chunk cache, I added a unit test but marking it as skipped due to a known issue. I have tested it locally.

### CODE REVIEW

**Comment by zhengbuqian** [comment]
e2e blocked by https://github.com/milvus-io/milvus/issues/33268

**Comment by zhengbuqian** [comment]
ut in test_sealed.cpp is skipped due to https://github.com/milvus-io/milvus/issues/33210

File: internal/core/src/mmap/Column.h
Comment (bigsheeper) [inline internal/core/src/mmap/Column.h:131]: madvise here may lead to memory increase. Perhaps it's best not to add it for now.

File: internal/core/src/mmap/Column.h
Comment (bigsheeper) [inline internal/core/src/mmap/Column.h:170]: any difference compared to the Data()?

Comment (zhengbuqian) [reply internal/core/src/mmap/Column.h:170]: `Data()` points at an array of elements, while `MmappedData()` points at the entire mmap-ed memory. They are the same for dense vectors but not the same for sparse vectors. `SparseFloatColumn` overrides `Data()` to return something different.

File: internal/core/src/mmap/Column.h
Comment (zhengbuqian) [reply internal/core/src/mmap/Column.h:131]: ack, will remove it before merging.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**
